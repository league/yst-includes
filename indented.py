# Test

print "Hello."
if x > 5:
    print "Indent."
print "Goodbye."
