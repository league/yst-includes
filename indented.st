
1. This is a list.

1. Here is another list item.

    ~~~~ {include="indented.py"}
    This pertains to previous list item.
    ~~~~

    A second paragraph of item 2.

1. The third list item.

