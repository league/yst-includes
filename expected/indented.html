<ol style="list-style-type: decimal">
<li><p>This is a list.</p></li>
<li><p>Here is another list item.</p>
<pre include="indented.py"><code># Test

print &quot;Hello.&quot;
if x &gt; 5:
    print &quot;Indent.&quot;
print &quot;Goodbye.&quot;</code></pre>
<p>A second paragraph of item 2.</p></li>
<li><p>The third list item.</p></li>
</ol>
